src/common/hook/useQueue/utils/sample-config.json

```diff
@@ -95,7 +95,7 @@
     }
   ],
   "documentStorage": {
-    "apiKey": "randonKey",
-    "url": "https://api-ropsten.tradetrust.io/storage"
+    "apiKey": "randomKey",
+    "url": "https://api-goerli.tradetrust.io/storage"
   }
```

Comment:
maybe we should signal to users this is supposed to be their own setup. let's not put any "correct" looking url? ok to leave `url` to say for eg, `https://api.example.com/storage`?